linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java

```diff
@@ -104,6 +104,12 @@ public void setup() throws IOException {
                 "(Lcom/google/protobuf/Message;)Lio/grpc/MethodDescriptor$Marshaller;",
                 false));
   }
+  
+  @Test
+  public void testCommandLine() {
+    String[] arguments = {"something"};
+    LinkageMonitor.parseCommandLine(arguments);
```

Comment:
How about checking the return value to ensure the code below (at line 96 of LinkageMonitor.java) works fine?

```
    CommandLine commandLine = parseCommandLine(arguments);
    String bomCoordinates = commandLine.getArgList().get(0);
```

Linkage Monitor is invoked with the versionless coordinates. ([doc](https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/linkage-monitor#installation))

```
java -jar linkage-monitor-X.Y.Z-all-deps.jar com.google.cloud:libraries-bom
```